src/v/features/feature_table.cc

```diff
@@ -686,10 +716,30 @@ void feature_table::set_license(security::license license) {
     _license = std::move(license);
 }
 
+void feature_table::set_builtin_trial_license(
+  model::timestamp cluster_creation_timestamp) {
+    if (ss::this_shard_id() == 0) {
+        vlog(featureslog.debug, "Initializing builtin trial license.");
+    }
```

Comment:
Is this conditional just for the log message?

Reply:
Yes, to only log on shard 0 and not all cores

Reply:
Maybe log the timestamp as well